app/models/user.rb

```diff
@@ -6,6 +6,8 @@ class User < ApplicationRecord
             include DeviseTokenAuth::Concerns::User
   has_many :todo_lists, dependent: :destroy
 
+
```

Comment:
in user model also we can have one more relationship to get all the collaborated todolists

Reply:
1)yes we need to add has_many assiciation with collaborator: which means: where all users can also be collaborator.
2) here we got to use collaboeator table as middle man to join the tododlist and user table to extract the tododlist of all the collaborator.
  has_many :collaborated_todo_lists, through: :collaborators, source: :todo_list